public/js/index.js

```diff
@@ -209,6 +209,10 @@ function initInstall() {
         return;
     }
 
+    $('#db_host').val("127.0.0.1:3306");
+    $('#db_user').val("root");
```

Comment:
Because of the default user for the database is root,I think root is a better choice.